list-navigation/lambda/providers/ddb-list-provider.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: LicenseRef-.amazon.com.-AmznSL-1.0
+// Licensed under the Amazon Software License  http://aws.amazon.com/asl/
+
+import { ListProvider, ListProviderRegistry, Page, PageToken } from '../list-provider';
+
+import { DynamoDBClient, ScanCommand, ScanCommandInput } from "@aws-sdk/client-dynamodb";
+import { marshall, unmarshall } from "@aws-sdk/util-dynamodb";
+import * as _ from 'lodash';
+
+export class DDBListProvider<T> implements ListProvider<T> {
```

Comment:
Can you add some comments to this describing what it is and how to use it? (especially what configuration needs to be provided)

Reply:
Yes! This PR was mostly to verify the code changes. I'll make the required comments and ReadMe changes in the upcoming PRs.